tests/testutils/ctx.go

```diff
@@ -220,6 +221,19 @@ func (ctx *RktRunCtx) rktBin() string {
 	return abs
 }
 
+func (ctx *RktRunCtx) GetUidGidRktBinOwnerNotRoot() (int, int) {
+	s, _ := os.Stat(ctx.rktBin())
```

Comment:
This may error out for some reason, in which case we implicitly panic here below. Better to check for error and fallback to GetUnprivilegedUidGid() in that case.